neo/SmartContract/SmartContractEvent.py

```diff
@@ -231,11 +229,18 @@ def __init__(self, event_type, event_payload, contract_hash, block_number, tx_ha
                     self.amount = int(BigInteger.FromBytes(event_payload[3])) if isinstance(event_payload[3], bytes) else int(event_payload[3])
                     self.is_standard_notify = True
 
-                elif plen == 3 and self.notify_type == NotifyType.REFUND:
+                elif self.notify_type == NotifyType.REFUND and plen >= 3:  # Might have more arguments
                     self.addr_to = UInt160(data=self.event_payload[1]) if len(self.event_payload[1]) == 20 else empty
                     self.amount = int(BigInteger.FromBytes(event_payload[2])) if isinstance(event_payload[2], bytes) else int(event_payload[2])
                     self.addr_from = self.contract_hash
                     self.is_standard_notify = True
+
+                elif self.notify_type == NotifyType.MINT and plen == 3:
+                    self.addr_to = UInt160(data=self.event_payload[1]) if len(self.event_payload[1]) == 20 else empty
+                    self.amount = int(BigInteger.FromBytes(event_payload[2])) if isinstance(event_payload[2], bytes) else int(event_payload[2])
+                    self.addr_from = self.contract_hash
```

Comment:
Is this the correct approach? I know that there isn't a defined standard around minting yet, but many smart contracts use a `null` `from` address for the minting operations. I realize this is just for the mint SC event notification, but I do wonder if this would be expected for someone writing an event handler. Probably not a big deal, but I thought I'd raise it as a consideration anyway...